src/components/templates/ExternalExperience/ExternalExperience.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect } from "react";
+import { useCss } from "react-use";
+import cn from "classnames";
+
+import { STRING_SPACE } from "settings";
+
+import { AnyVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+import { externalUrlAdditionalProps, openUrl } from "utils/url";
+
+import { RenderMarkdown } from "components/organisms/RenderMarkdown";
+
+import CN from "./ExternalExperience.module.scss";
+
+const componentRules = {
+  p: "span",
+};
+
+interface ExternalExperienceProps {
+  space: WithId<AnyVenue>;
+}
+
+export const ExternalExperience: React.FC<ExternalExperienceProps> = ({
+  space,
+}) => {
+  const redirectUrl = space.zoomUrl;
+
+  useEffect(() => {
+    if (!redirectUrl) return;
+
+    openUrl(redirectUrl);
+  }, [redirectUrl]);
+
+  const styles = useCss({
+    background: `url(${space.config?.landingPageConfig.coverImageUrl}) center center`,
```

Comment:
Not necessary to check if image exists?
Also, some of these sausages can be shortened by `const landingConfig = space.config?.langindPageConfig`

Reply:
done https://github.com/sparkletown/sparkle/pull/3015/commits/42838ddbe607e71b4b4b587e9c83c1883992155b